apps/tlon-mobile/src/App.tsx

```diff
@@ -54,12 +51,10 @@ const App = ({ wer: initialWer }: Props) => {
   useDevTools({ enabled: DEV_LOCAL, localCode: DEV_LOCAL_CODE });
```

Comment:
I know this isn't part of your changes, but could you add a comment here indicating 1) What the `wer` is and 2) Where it's being passed in from? Since it's not clear from the context.